### ISSUE TITLE
Ticket Object Manager Greyed Out - Not Editable

### ISSUE THREAD

**Issue by tdavison1** [zammad/zammad#1525]
### Infos:

* Used Zammad version: 2.0.x
* Used Zammad installation source: CentOS via RPM
* Operating system: CentOS 7 (all patches)
* Browser + version: Mac High Siera & Windows 7  Firefox 56.0

### Expected behavior:

* Ability to edit the objects (eg; Group) to set them as required when filling out a ticket.

### Actual behavior:

* The objects are not editable. I can however edit any objects that I have created, just not those created by a default install. I have tried this as a LDAP user and also full default admin user. The only editable item is Type - all the others are in the `class="is-grayed-out u-notAllowed"`
![image](https://user-images.githubusercontent.com/32715403/31457139-c6290c36-aeb3-11e7-998a-c41903278fdc.png)

### Steps to reproduce the behavior:

* Base installation of Centos 7 with the install as per - https://docs.zammad.org/en/latest/install-centos.html - Login, go to Admin -> Objects -> all items greyed out.

**Comment by tdavison1** [comment]
Just tried another install on blank Centos 7 and Ubuntu with the same results.

**Comment by hanneshal** [comment]
Thats correct and the desired behaviour. Zammad needs some basics to rely on, these fields are part of this. 
Which of those attributes do you want to edit ?

**Comment by tdavison1** [comment]
Hi,

We absolutely want to keep these fields, we just want to be able to change the screens to make them mandatory fields for example;
<img width="591" alt="image" src="https://user-images.githubusercontent.com/32715403/31713904-13083e68-b400-11e7-8406-4383ad5cb913.png">

By default they are not shown or mandatory. Is this something that is possible?

Thanks

**Comment by tdavison1** [comment]
I have upgraded to V2.1 release with the same results. 

Is there another way to make these fields shown on the ticket forms?

Thanks

**Comment by tbsfnk** [comment]
Any news on this?

**Comment by MrGeneration** [comment]
Sorry but I'm locking the conversation to contributors only.
Please don't ping/bump issues on this repository as it creates a lot of noise.

We will update issues accordingly as soon as anything moves.
This issue is a feature backlog and has no roadmap yet.
